### PR TITLE
feat: Make it clear that voting power includes CAKE before XX block

### PR DESCRIPTION
**Pull request by Chef-Yogi**
Close PAN-207

before:
before

after:
after

### CODE REVIEW

File: src/config/localization/translations.json [hunk 1]
```diff
@@ -1527,5 +1527,6 @@
   "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.": "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.",
   "Current Market Price": "Current Market Price",
   "Limit Price": "Limit Price",
-  "Error Tracking Id": "Error Tracking Id"
+  "Error Tracking Id": "Error Tracking Id",
+  "YOUR CAKE HELT AT BLOCK": "YOUR CAKE HELT AT BLOCK"
```
Comment (chef-jojo) [inline src/config/localization/translations.json:1531]: HELT? 😄

Comment (Chef-Yogi) [reply src/config/localization/translations.json]: fixed

**Review by chef-jojo — APPROVED** [review]

File: src/config/localization/translations.json [hunk 2]
```diff
@@ -1527,5 +1526,6 @@
   "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.": "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.",
   "Current Market Price": "Current Market Price",
   "Limit Price": "Limit Price",
-  "Error Tracking Id": "Error Tracking Id"
+  "Error Tracking Id": "Error Tracking Id",
+  "YOUR CAKE HELD AT BLOCK": "YOUR CAKE HELD AT BLOCK"
```
Comment (memoyil) [inline src/config/localization/translations.json:1530]: Use lowercase here and make it uppercase in jsx

File: src/views/Voting/components/CastVoteModal/DetailsView.tsx [hunk 3]
```diff
@@ -41,7 +53,10 @@ const DetailsView: React.FC<DetailsViewProps> = ({
         </Text>
       </VotingBox>
       <Text color="secondary" textTransform="uppercase" mb="4px" bold fontSize="14px">
```
Comment (memoyil) [inline src/views/Voting/components/CastVoteModal/DetailsView.tsx:55]: This already makes the text uppercase so translation can be lowercase

File: src/config/localization/translations.json (hunk 2 above)
Comment (Chef-Yogi) [reply src/config/localization/translations.json:1530]: thanks for reminding!

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**